python/dask_cudf/dask_cudf/io/json.py

```diff
@@ -2,8 +2,11 @@
 
 from functools import partial
 
-import dask
+import dask.dataframe as dd
 
 import cudf
 
-read_json = partial(dask.dataframe.read_json, engine=cudf.read_json)
+read_json = partial(dd.read_json, engine=cudf.read_json)
+read_json_experimental = partial(
+    dd.read_json, engine=partial(cudf.read_json, engine="cudf_experimental")
+)
```

Comment:
What are the risks of using `engine="cudf_experimental"` in place of `engine="cudf"` everywhere?

Reply:
To clarify: I don't think any users will use this API or specify `engine="cudf_experimental"` themselves. So, we really need move `engine="cudf_experimental"` functionality into `engine="cudf"` as soon as we can.

Reply:
@GregoryKimball